code/game/objects/items/RPD.dm

```diff
@@ -22,22 +22,28 @@ GLOBAL_LIST_INIT(atmos_pipe_recipes, list(
 	),
 	"Devices" = list(
-		new /datum/pipe_info/pipe("Connector",			/obj/machinery/atmospherics/components/unary/portables_connector),
-		new /datum/pipe_info/pipe("Unary Vent",			/obj/machinery/atmospherics/components/unary/vent_pump),
-		new /datum/pipe_info/pipe("Gas Pump",			/obj/machinery/atmospherics/components/binary/pump),
-		new /datum/pipe_info/pipe("Passive Gate",		/obj/machinery/atmospherics/components/binary/passive_gate),
-		new /datum/pipe_info/pipe("Volume Pump",		/obj/machinery/atmospherics/components/binary/volume_pump),
-		new /datum/pipe_info/pipe("Scrubber",			/obj/machinery/atmospherics/components/unary/vent_scrubber),
-		new /datum/pipe_info/pipe("Injector",			/obj/machinery/atmospherics/components/unary/outlet_injector),
-		new /datum/pipe_info/pipe("Passive Vent",		/obj/machinery/atmospherics/components/unary/passive_vent),
+		new /datum/pipe_info/pipe("Connector",					/obj/machinery/atmospherics/components/unary/portables_connector),
+		new /datum/pipe_info/pipe("Gas Pump",					/obj/machinery/atmospherics/components/binary/pump),
+		new /datum/pipe_info/pipe("Volume Pump",				/obj/machinery/atmospherics/components/binary/volume_pump),
+		new /datum/pipe_info/pipe("Gas Filter",					/obj/machinery/atmospherics/components/trinary/filter),
+		new /datum/pipe_info/pipe("Gas Mixer",					/obj/machinery/atmospherics/components/trinary/mixer),
+		new /datum/pipe_info/pipe("Passive Gate",				/obj/machinery/atmospherics/components/binary/passive_gate),
+		new /datum/pipe_info/pipe("Injector",					/obj/machinery/atmospherics/components/unary/outlet_injector),
+		new /datum/pipe_info/pipe("Scrubber",					/obj/machinery/atmospherics/components/unary/vent_scrubber),
+		new /datum/pipe_info/pipe("Large Scrubber",				/obj/machinery/atmospherics/components/unary/vent_scrubber/high_volume),
+		new /datum/pipe_info/pipe("Unary Vent",					/obj/machinery/atmospherics/components/unary/vent_pump),
+		new /datum/pipe_info/pipe("Large Unary Vent",			/obj/machinery/atmospherics/components/unary/vent_pump/high_volume),
+		new /datum/pipe_info/pipe("Passive Vent",				/obj/machinery/atmospherics/components/unary/passive_vent),
+		new /datum/pipe_info/pipe("Manual Valve",				/obj/machinery/atmospherics/components/binary/valve),
+		new /datum/pipe_info/pipe("Digital Valve",				/obj/machinery/atmospherics/components/binary/valve/digital),
 		new /datum/pipe_info/meter("Meter"),
-		new /datum/pipe_info/pipe("Gas Filter",			/obj/machinery/atmospherics/components/trinary/filter),
-		new /datum/pipe_info/pipe("Gas Mixer",			/obj/machinery/atmospherics/components/trinary/mixer),
-		new /datum/pipe_info/pipe("Pressure Valve",		/obj/machinery/atmospherics/components/binary/pressure_valve),
-		new /datum/pipe_info/pipe("Temperature Gate",	/obj/machinery/atmospherics/components/binary/temperature_gate),
-		new /datum/pipe_info/pipe("Temperature Pump",	/obj/machinery/atmospherics/components/binary/temperature_pump),
```

Comment:
You appear to be removing the pressure valve, temperature gate and temperature pump from the RPD.

Reply:
Fixed